DirectEmployers/MyJobs
# [PD-3278] add timespan based options

## What's this PR do?
PR adds the timeline chart into Analytics

## Screenshots (if appropriate)
![screen shot 2017-06-01 at 11 49 15 am](https://cloud.githubusercontent.com/assets/22029077/26688366/6b47edb0-46c0-11e7-817c-a1da52129345.png)


## Where should the reviewer start?
Start inside of any of the analytics reports

## How should this be manually tested?
Pull down the repository. Go into any of the analytics apps and click on the day, week, or month field and you should see the timeline drop down with the data inside of it

## Any background context you want to provide?
Will need some extensive testing to make sure the timeline and data coming back are correct
